src/i18n/locale/en.json

```diff
@@ -128,11 +130,17 @@
   "components.Settings.Notifications.enableSsl": "Enable SSL",
   "components.Settings.Notifications.save": "Save Changes",
   "components.Settings.Notifications.saving": "Saving…",
+  "components.Settings.Notifications.settinguptelegram": "Setting up Telegram Notifications",
+  "components.Settings.Notifications.settinguptelegramDescription": "To setup Telegram you need to <CreateBotLink>create a bot</CreateBotLink> and get the bot API key.    Additionally, you need the chat id for the chat you want the bot to send notifications to.    Which you can do by adding <GetIdBotLink>@get_id_bot</GetIdBotLink> to the chat or group chat.",
```

Comment:
Don't forget your change after running the extract won't update this file. You need to do this one manually now.

Reply:
You noticed. Nevermind! 😆